Firestore/core/src/model/value_util.h

```diff
@@ -155,6 +176,15 @@ google_firestore_v1_Value MaxValue();
  */
 bool IsMaxValue(const google_firestore_v1_Value& value);
 
+/**
+ * Returns `true` if `value` represents a VectorValue..
+ */
+bool IsVectorValue(const google_firestore_v1_Value& value);
+
+int64_t IndexOfKey(const google_firestore_v1_MapValue& mapValue,
```

Comment:
Can you add some comment for this function.

Reply:
Added. Thanks.